CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Unreleased
 
+# 2.0.3
+- Make crate MSRV `1.63`.
```

Comment:
```suggestion
- Document crate MSRV of `1.63`.
```
It already is 1.63 since v2.0.2.

Reply:
You're right, that wordage is fine with me.